src/types/resources.ts

```diff
@@ -90,6 +93,7 @@ export interface ResourceState {
   [ResourceType.Dashboards]: DashboardsState
   [ResourceType.Labels]: NormalizedState<Label>
   [ResourceType.Members]: NormalizedState<Member>
+  [ResourceType.Notebooks]: NotebooksState
```

Comment:
I'm confused on what we are doing on a higher level. Is the goal to:
(a) move the flows state management model into redux?
     or
(b) do existence checks on the presence of notebooks, to determine if the user has that feature depreciated or not?

Because if it's the latter...I'm not sure we want to populate resources for tracking of feature depreciation. It's a lot of stuff for a simple boolean response, which may be better consolidated in a `state.depreciation.<map>` with the boolean calculated on the single fetch (on load of org user).

Does this make sense?

Reply:
@wiedld it's the latter. We're planning on deprecating notebooks, and the first part of that is hiding notebooks depending on rules listed in the [linked ticket.](https://github.com/influxdata/idpe/issues/16723)

My thinking here is that the response that comes back is small - it's essentially a date, an id, and a name. On prod, I fetched 4 notebooks and the total size was 1.37 kB, see attached screenshot. Since it's so small and trivial to store in memory, I figured that it might be better to just store the notebooks themselves in case they might be useful for the deprecation of this feature (maybe we need to include a nag that says "We're deprecating notebooks soon, and you have 13 notebooks. Please follow this link to start transferring them" or something like that).

If I was going to do it right, I'd create a new backend route that just returns a count of notebooks, but doing that would require making changes in at least two repos and I want to move quickly on this, since we have other unknown work to do like making our visualizations work with SQL without a lot of work.

![Screen Shot 2022-12-19 at 10 02 27 AM](https://user-images.githubusercontent.com/146112/208455604-3d614a3e-73d0-4670-bd78-ee4cc9d575fc.png)

Reply:
Due to our time differences and me wanting to finish this up, I'm going to move forward with this. If you have strong opinions against it, we can follow up with a tweak to how this is done

Reply:
No worries. Glad you moved ahead and I didn't block it!